server/main.lua

```diff
@@ -143,7 +113,11 @@ RegisterNetEvent('qb-cityhall:server:ApplyJob', function(job)
     TriggerClientEvent('QBCore:Notify', src, Lang:t('info.new_job', {job = JobInfo.label}), 'success')
 end)
 
-RegisterNetEvent('qb-cityhall:server:getIDs', giveStarterItems)
+RegisterNetEvent('qbx-core:server:createCharacter', function()
+    local src = source
+    if not src then return end
```

Comment:
net event will always have source if called from the client, no need to check for its existence.

Also secure this net event please :)
https://manason.github.io/effective-fivem-lua/events/#secure-net-events

Reply:
Sounds good, thanks!